pkg/plugin/conntrack/_cprog/conntrack.c

```diff
@@ -71,6 +76,13 @@ struct {
     __uint(pinning, LIBBPF_PIN_BY_NAME); // needs pinning so this can be access from other processes .i.e debug cli
 } retina_conntrack SEC(".maps");
 
+/**
+ * Helper function to check if a packet sequence number is increasing.
+ */
+static inline bool is_seq_increasing(__u32 seq1, __u32 seq2) {
```

Comment:
is there a reason why we can't just compare the sequence number2?

Reply:
This is to handle the case where sequence numbers wrap around.